src/pages/docs/salesforce-testing/special-nlps.md

```
@@ -368,4 +375,16 @@ See the GIF below demonstrating how to use this NLP.
 
 ![Get Record Details](https://s3.amazonaws.com/static-docs.testsigma.com/new_images/projects/applications/SF_Read_API.gif)
 
+---
+
+## **NLP 18:**
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)